sdk/schema/fieldname/names.go

```diff
@@ -49,6 +50,7 @@ const (
 	Token           = sdk.FieldName("Token")
 	URL             = sdk.FieldName("URL")
 	User            = sdk.FieldName("User")
+	User_Access_Token = sdk.FieldName("User Access Token")
```

Comment:
nit: We didn't use the underline `_` convention so far in naming these, so let's rephrase this to `UserAccessToken`

Reply:
Thanks, Updated this change